src/test/java/core/model/TranslatorTest.java

```diff
@@ -0,0 +1,207 @@
+package core.model;
+
+import org.apache.commons.lang3.ArrayUtils;
+import org.apache.commons.lang3.StringUtils;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.List;
+import java.util.Locale;
+import java.util.MissingResourceException;
+import java.util.stream.Stream;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.assertj.core.api.AssertionsForClassTypes.assertThatThrownBy;
+import static org.junit.jupiter.params.provider.Arguments.of;
+
+class TranslatorTest {
+
+    private static final List<String> SUPPORTED_LANGAUGES = List.of(
```

Comment:
Typo, `LANGUAGES` (a few more occurrences in this file)

Reply:
Review Finding 3